tests/eas/load_tracking.py

```diff
@@ -1273,3 +1290,224 @@ def test_util_tg4_group(self):
+                          /       \
+                      +------+   +------+
+                      |"/tg1"|   |"/tg2"|
+                      +------+   +------+
+                       /          /    \
+                      /          /      \
+                    t1_1       t1_2    t2_1
+    """
+    target_cpu = 0
+    root_group = None
+    trace = None
+
+    @classmethod
+    def _getExperimentsConf(cls, test_env):
+        # Run all workloads on a CPU with highest capacity
+        cls.target_cpu = [min(test_env.calibration(),
+                              key=test_env.calibration().get)]
+        period_ms = 16
+        duration_s = 4
+        # Create taskgroups
+        cpus = test_env.target.list_online_cpus()
+        cls.root_group = Taskgroup("/", cpus, 0, test_env)
+        tg1 = Taskgroup("/tg1", cpus, 0, test_env)
+        tg2 = Taskgroup("/tg2", cpus, 0, test_env)
+
+        # Create tasks
+        t1_1 = Task("t1_1", test_env, cls.target_cpu, period_ms=period_ms,
+                    duty_cycle_pct=10, duration_s=duration_s)
+        t1_2 = Task("t1_2", test_env, cls.target_cpu, period_ms=period_ms,
+                    duty_cycle_pct=20, duration_s=duration_s)
+        t2_1 = Task("t2_1", test_env, cls.target_cpu, period_ms=period_ms,
+                    duty_cycle_pct=5, duration_s=duration_s)
+
+        # Link nodes to the hierarchy tree
+        cls.root_group.add_children([tg1, tg2])
+        tg1.add_children([t1_1, t1_2])
+        tg2.add_children([t2_1])
+
+        conf = {
+            'tag' : 'cgp_migration',
+            'flags' : ['ftrace', 'freeze_userspace'],
+            'cpufreq' : {'governor' : 'performance'},
+        }
+
+        return {
+            'wloads': {},
+            'confs': [conf],
+        }
+
+    @classmethod
+    def setUpClass(cls, *args, **kwargs):
+        super(CgroupsMigrationTest, cls).runExperiments(*args, **kwargs)
+
+    @classmethod
+    def _migrate_task(cls, test_env):
+        exclude = ''
+        cgroups = test_env.target.cgroups
+
+        tg1_tasks = cgroups.controllers['cpu'].tasks('/tg1')
+        for pid, (task_name, cmdline) in tg1_tasks.iteritems():
+            # Get the PID of the task t1_1 which is not migrated
+            if task_name == 'rt-app' and re.search(r't1_1', cmdline):
+                exclude = '{}'.format(pid)
+                break
+
+        # Migrate t1_2 to the group /tg2 without migrating t1_1
+        task = cls.root_group.get_child('t1_2')
+        new_taskgroup = cls.root_group.get_child('/tg2')
+        task.change_taskgroup(new_taskgroup)
+        srcg = cgroups.controllers['cpu'].cgroup('/tg1').directory
+        dstg = cgroups.controllers['cpu'].cgroup('/tg2').directory
+        cgroups.cgroups_tasks_move(srcg, dstg, exclude=exclude)
+
+    def test_util_root_group(self):
+        """Test utilization propagation to cgroup root"""
+        return self._test_group_util('/')
+
+    def test_util_tg1_group(self):
```

Comment:
Thinking about the real meaning of these tests.... maybe if we find a more descriptive name it can be easier to understand what is failing when something is failing...

Thus, what about a renaming similar to:
- `s/test_util_root_group/test_util_aggregation/`
- `s/test_util_tg1_group/test_util_move_out/`
- `s/test_util_tg2_group/test_util_move_in/`

Reply:
Better proposals are more then welcome... and the test docstring should also be update to better describe the tested condition. For example, for the last test, maybe something like:

```python
def test_util_move_in(self):
"""
Test utilization when a task enter a group.
"""
```

Reply:
Done